conda-forge.yml

```diff
@@ -1,4 +1,6 @@
 conda_forge_output_validation: true
+bot:
+  inspection: update-grayskull
```

Comment:
The one hesitation that I have here is that having bot updates to the recipe also tried to bring in optional dependencies, which is why `pyhf` dropped down to `inspection: hint`. c.f. https://github.com/conda-forge/pyhf-feedstock/pull/22

Though if you think this is fine then I'm okay with it.

Reply:
We could give it a try. Is `update-grayskull` different from `update-all`?

Reply:
> We could give it a try.

:+1: 

> Is `update-grayskull` different from `update-all`?

@henryiii Yes. According to https://conda-forge.org/docs/maintainer/conda_forge_yml/#bot

```yaml
    # The bot.inspection key in the conda-forge.yml can have one of seven possible values and controls
    # the bots behaviour for automatic dependency updates:
    inspection: hint  # generate hints using source code (backwards compatible)
    ...
    inspection: update-all  # update recipe using all methods
    inspection: update-source  # update recipe using only source code
    inspection: update-grayskull  # update recipe using only grayskull
```

so I think(?) `update-all` is just the combination of `update-source` and `update-grayskull`.

Reply:
What's update-source? Do we lose something?

Reply:
@henryiii I'm not really sure. I think this is a @chrisburr question.